src/systems/filecoin_vm/sysactors/register.go

```diff
@@ -0,0 +1,26 @@
+package sysactors
+
+import vmr "github.com/filecoin-project/specs/systems/filecoin_vm/runtime"
+
+// init is called in Go during initialization of a program.
+// this is an idiomatic way to do this. Implementations should approach this
+// howevery they wish. The point is to initialize a static registry with
+// built in pure types that have the code for each actor. Once we have
+// a way to load code from the StateTree, use that instead.
+func init() {
+	_registerBuiltinActors()
+}
+
+func _registerBuiltinActors() {
+	// TODO
+
+	cron := &CronActorCode_I{}
+
+	vmr.RegisterActor(vmr.InitActorCodeCID, &InitActorCode_I{})
+	vmr.RegisterActor(vmr.CronActorCodeCID, cron)
```

Comment:
Do the storage power and market actors need to be registered too?

Reply:
Yes -- this function was moved from elsewhere so I didn't edit the content in the initial PR, but I've just added those entries.